core/storage/exploration/gae_models_test.py

```diff
@@ -532,24 +532,22 @@ def test_get_all_non_private_commits(self):
         private_commit.put()
         public_commit.update_timestamps()
         public_commit.put()
-        results, _, more = (
+        results, _, _ = (
```

Comment:
See previous notes on `more`, here and below.

Reply:
Done.